SpiralCompiler/Program.cs

```diff
@@ -1,10 +1,19 @@
-﻿namespace SpiralCompiler
+﻿using SpiralCompiler.Syntax;
+
+namespace SpiralCompiler
```

Comment:
Suggestion, consider using file-scoped namespaces, reduces indentation by a lot.
VS can convert the entire project for you.